### PR TITLE
Create installation package

### PR DESCRIPTION
**Pull request by mflueggen**
Requires build-essential, devscripts and debhelper version 9 or higher (see https://wiki.debian.org/Packaging/Intro?action=show&redirect=IntroDebianPackaging for an example)

Run dpkg-buildpackage -us -uc in deploy/yats-1.0 to create dep package.

### CODE REVIEW

**Review by lawben — APPROVED** [review]
looks good

File: deploy/yats-1.0/debian/copyright [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+It was downloaded from:
+
+    https://github.com/hpicgs/yats
+    
+Upstream Author:
+
+    Lawrence Benson <lawrence.benson@student.hpi.de>
+    Jan van Dieken <jan.dieken@student.hpi.de>
+    Mathias Flueggen <mathias.flueggen@student.hpi.de>
+    Konstantin Harmuth <konstantin.harmuth@student.hpi.de>
```
Comment (Highlife1911) [inline deploy/yats-1.0/debian/copyright]: Is it required to write the email here?

Comment (lawben) [reply deploy/yats-1.0/debian/copyright]: maybe just the github name?

Comment (mflueggen) [reply deploy/yats-1.0/debian/copyright]: Willy had it in his example. I'll just put "YaTS Team"

**Comment by Highlife1911** [comment]
yes this works

**Pull request merged**